src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.html

```diff
@@ -36,14 +36,22 @@
                 (click)="reset()">
           <i [ngClass]="[icons.undo]"></i>
         </button>
+        <button class="btn btn-light my-1 ml-3"
+                i18n-title
+                title="Information"
```

Comment:
Will this be the tooltip? We should try to provide a descriptive text of what this button does before pushing it...

Reply:
Thanks, I have changed this to "show hidden information"